common/customproperties.go

```diff
@@ -32,23 +32,29 @@ func (c CustomProperties) X() *custom_properties.Properties {
 	return c.x
 }
 
+// PropertiesList returns the list of all custom properties of the document.
 func (c CustomProperties) PropertiesList() []*custom_properties.CT_Property {
 	return c.x.Property
 }
 
-func (c CustomProperties) GetPropertyByName(name string) CustomProperty {
-	property := c.getPropertyByName(name)
-	if property == nil {
-		return nil
+// GetCTPropertyByName returns a custom property selected by it's name.
+func (c CustomProperties) GetCTPropertyByName(name string) *custom_properties.CT_Property {
+	propsList := c.x.Property
+	for _, property := range propsList {
+		if *property.NameAttr == name {
+			return property
+		}
 	}
-	return CustomProperty(property)
+	return nil
 }
 
-func (c CustomProperties) getPropertyByName(name string) *custom_properties.CT_Property {
+// GetPropertyByName returns a custom property selected by it's name.
+// This method is deprecated and will be removed in the future.
```

Comment:
Deprecated notice should be formatted with "Deprecated: ..."  Similar to other locations
Instead use

```go
// GetPropertyByName returns a custom property selected by it's name.
// Deprecated: Use GetCTPropertyByName.  Will be removed in next major version (v2).
```